Wiki.md

```diff
@@ -177,3 +177,12 @@ If the deployment of changes to the wiki following the merging of a pull request
 1. Navigate to the Oppia wiki repository's Actions tab.
 2. Select the "Deploy to wiki" workflow.
 3. Refer to these [instructions](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow?tool=webui#running-a-workflow) to manually execute the workflow.
+
+## Tips and Tricks
+
```

Comment:
1 final nit. Please add this to the table of contents on top of the page. Thanks!

Reply:
Done @gp201